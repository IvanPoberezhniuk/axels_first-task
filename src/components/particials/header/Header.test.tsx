import { shallow, ShallowWrapper } from 'enzyme';

import { Header } from '../../index';

let component: ShallowWrapper;
const setUp = () => shallow(<Header />);

beforeEach(() => {
  component = setUp();
});

describe('Header component', () => {
  it('should take snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});

