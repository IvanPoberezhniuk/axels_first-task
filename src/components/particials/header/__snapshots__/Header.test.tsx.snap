// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component SNAPSHOT 1`] = `
<styled.header>
  <h1>
    TEST APP
  </h1>
</styled.header>
`;
